images/base/Dockerfile.arch

```diff
@@ -28,6 +28,10 @@ RUN pacman --noconfirm -Syyuu \
 # Enables Docker starting with systemd
 RUN systemctl enable docker
 
+# Add docker-compose
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Can you use a [build argument](https://docs.docker.com/engine/reference/builder/#arg) for the version number, so that we can update it easily? Something like this:

```Dockerfile
ARG COMPOSE_VERSION=1.29.2
RUN curl -L "https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Also, combine the RUN lines together, because each of [these will create a layer](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) and doing this in two steps doubles the disk usage. To explore the resulting docker image, you can use a tool like: https://github.com/wagoodman/dive